### DISCUSSION TITLE
パッドが剥がれてしまい、LEDがつかない件

### DISCUSSION THREAD

**Discussion by ys3124** [Yowkees/keyball#247] [Q&A]
先日keyball44を購入させていただき、組み立てを進めております。

LEDの半田付け時に画像赤で囲った二箇所のパッド？を剥がしてしましました。
LEDが光るよう対処することは可能でしょうか？
可能であれば方法を教えていただけますと幸いです。
![IMG_9864](https://github.com/Yowkees/keyball/assets/36505403/aae2e93b-6088-41c1-829c-649e5d9b9006)

**Comment by Yowkees** [comment]
![KB44LEDfix](https://github.com/Yowkees/keyball/assets/84209772/f19e9ec0-b2b5-456d-904a-5528d07d54a1)
ご連絡ありがとうございます。パッド剥がれ了解しました。
添付画像の黄色と緑の同色同士の点をエナメル線やゴム被覆導線でショートさせてください。
これでLEDを復活できると思います。
お手数ですがご対応お願いいたします。

**Reply by ys3124** [reply]
無事に全てのLEDを光らせることができました。
ありがとうございました！

**Comment by Yowkees** [comment]
無事に直って良かったです！
